source/client/BUILD

```diff
@@ -49,3 +47,20 @@ envoy_cc_library(
         "@envoy//source/exe:process_wide_lib",
     ],
 )
+
+envoy_cc_library(
```

Comment:
I'm still really unfamiliar with Nighthawk, so maybe this would make more sense to me with a quick walkthrough, but I'm not clear on why "service" stuff is living in /source/client. Similar with the class/namespace names, e.g. `Nighthawk::Client::ServiceMain`.

Reply:
sgtm; add a TODO for that too?

Reply:
Filed https://github.com/envoyproxy/nighthawk/issues/96